docs/user-guide/mvd-configuration.md

```diff
@@ -292,7 +292,90 @@ cp "//'[output_dataset_name]'" 'zlux-app-server/deploy/instance/ZLUX/serverConfi
+
+Plug-ins can also have endpoints that control their [configuration](https://zowe.github.io/docs-site/latest/extend/extend-desktop/mvd-configdataservice.html#configuration-dataservice). These endpoints have scope (product/site/instance/user) and the data is stored and retrieved by path name.
+
+#### URLs
+
+Zowe service endpoints are identified by URLs that are formatted like this:
+
+`/<product>/plugins/<plugin id>/services/<service>/<version>/<path>`
+
+For example:
+
+`/ZLUX/plugins/org.zowe.foo/services/baz/_current/users/fred`
+
+Where product is `ZLUX`, plugin id is `org.zowe.foo`, service is `baz`, version is `_current`, and path is `/users/fred`.
+
+To access endpoints when RBAC is enabled, users must have READ access to a corresponding SAF profile in the ZOWE class==?==. SAF profile names have this format:
+
+`<product>.<instance id>.SVC.<pluginid_with_underscores>.<service>.<HTTP method>.<url with forward slashes '/' replaced by periods '.'>`
+
+For example, to issue a POST request to the service endpoint documented above using the default ==Zowe Application Server?== instance, users must have READ access to the following SAF profile:
+
+`ZLUX.DEFAULT.SVC.ORG_ZOWE_FOO.BAZ.POST.USERS.FRED`
+
+Note: "DEFAULT" is the default Zowe instance ID.
+
+Configuration endpoints have SAF profile names with this format:
+
+`<product>.<instance id>.**CFG.<pluginid_with_underscores>.<service>.<HTTP method>.<url with forward slashes '/' replaced by periods '.'>`
+
+==what's with the **CFG?==
+
+For example, using the default instance, users must have access to the following SAF profile to access the instance-scoped configuration element “files”:
+
+`ZLUX.DEFAULT.CFG.ORG_ZOWE_FOO.GET.INSTANCE.FILES`
+
+### Enabling RBAC
+
+By default, RBAC is disabled and all authenticated Zowe users can access all services. To enable RBAC, take the following steps:
+
+1. Open the Zowe Application Server configuration JSON file. In the default server instance, the configuration file is `/zlux-app-server/config/zluxserver.json`.
+2. In the `dataserviceAuthentication` object, add `"rbac": true`.
+
+### Configuring basic authorization
+
+We recommend configuring some basic authorization:
+
+- To give administrators access to everything in Zowe, create a `ZLUX.**` profile and give them UPDATE access to it.
+- To give non-administrators basic access to the site and product, create a `ZLUX.*.ORG_ZOWE_*`  profile and give them READ access to it.
+- To prevent non-administrators from configuring endpoints at the product and instance levels, create a ?? profile and give them no access to it.
+- To give non-administrators all access to user, create a ?? profile and give them ?? access to it.
+
+### Creating generic authorization
```

Comment:
I would mention this earlier, given that we've already recommended creating generic profiles (i.e. ones that contain * or **)